channels_irc/cli.py

```diff
@@ -101,19 +101,21 @@ def run(self, args):
             format="%(asctime)-15s %(levelname)-8s %(message)s",
         )
 
-        if not any([args.channel_layer, args.server, args.nickname]):
+        if not any([args.application, args.server, args.nickname]):
```

Comment:
I could be mistaken, but if as the error message says, `--application`, `--server`, and `--nickname` are required, then `any()` should be `all()`. `not any()` is only true if none of the arguments are `True`. So it seems like either the exception message or the function call is incorrect

```
>>> any([True, True, False])
True
>>> any([False, False, False])
False
>>> all([True, True, False])
False
```

Reply:
100% correct.  Good catch.